exercises/concept/freelancer-rates/freelancer_rates_test.cpp

```diff
@@ -0,0 +1,62 @@
+#include "freelancer_rates.cpp"
```

Comment:
Students seeing this will most likely find it incredibly strange. Since the code organization concept is a prerequisite, why not add a header file which declares the functions?

Reply:
The test file will look overwhelming for them even if they understand headers and namespaces. There is so much going on here. I want to ease the process as much as possible.

To be honest I felt the other way around and was really confused why everyone is using headers and nested namespaces on exercises to define two small functions. It felt almost "java-level-code-bloated" to me when I started out.